.github/workflows/e2e.yml

```diff
@@ -0,0 +1,30 @@
+name: E2E
+
+on:
+  pull_request:
+
+env:
+  GO_VERSION: '1.18'
+
+jobs:
+  e2eTests:
```

Comment:
nit: comment the cgroups version of the OS (i think is v1 but I would explicitly mentioned it for the future us) , might be as job name

Reply:
Nice. Changed!